docs/index.rst

```diff
@@ -42,6 +42,20 @@ alternatively::
 
     config.include('pyramid_storage.s3')
 
+To use GCS file storage instead of storing files locally on your server (the default assumption)::
+
+    pyramid.includes =
+        pyramid_storage.gcloud
+
+alternatively::
+
+    config.include('pyramid_storage.gcloud')
+
+We're supporting these authentication methods for GCS::
+
+* JSON credentials file (requires a credentials file to be deployed to the env you're running in, as well as the `credentials` argument pointing at the path.).
```

Comment:
nit: backquotes should be doubled in `rst` or they show up as italic/emphasize

Reply:
hey @leplatrem, thanks, updated that part :+1: